### DISCUSSION TITLE
Update procedure for pigallery2 installed from release as per point 1.2.1-a

### DISCUSSION THREAD

**Discussion by ratavaz** [bpatrik/pigallery2#320] [Q&A]
Hi all,
sorry for the newbie question.
I've managed to install pigallery2 version 1.8.0 on a Raspberry PI 3 following the "1.2.1-a Install from release" descriptions now I'd like to understand how to upgrade to the latest version.
Am I supposed to follow the same procedure described in "1.2.1-a Install from release" using the latest release and basically "reinstall" everything or it is enough to update some of the files from the release?

Thanks for your help
Paolo

**Comment by bpatrik** [comment]
Hi,

yeah similar steps should work. Basically get the sources and make sure you run `npm install`.
Also I would recommend using docker. Thinks are just simpler with that.
